src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripMenuItem.ToolStripMenuItemAccessibleObjectTests.cs

```diff
@@ -69,6 +69,86 @@ public void ToolStripMenuItemAccessibleObject_GetPropertyValue_ControlType_IsExp
             Assert.Equal(expected, actual);
         }
 
+        [WinFormsFact]
+        public void ToolStripMenuItemAccessibleObject_GetPropertyValue_PositionInSet_ReturnsExpected_IfNoParent()
+        {
+            using ToolStripMenuItem toolStripMenuItem = new();
+
+            AccessibleObject accessibilityObject = toolStripMenuItem.AccessibilityObject;
+
+            Assert.Null(accessibilityObject.GetPropertyValue(UiaCore.UIA.PositionInSetPropertyId));
+        }
+
+        [WinFormsFact]
+        public void ToolStripMenuItemAccessibleObject_GetPropertyValue_PositionInSet_ReturnsExpected()
+        {
+            using MenuStrip menuStrip = new();
+
+            ToolStripMenuItem item1 = new();
```

Comment:
```suggestion
            using ToolStripMenuItem item1 = new();
```
Same for `separator` and `item2` in this method and also in `ToolStripMenuItemAccessibleObject_GetPropertyValue_SizeOfSet_ReturnsExpected`.